.changeset/orange-cougars-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/techdocs-common': patch
```

Comment:
When you add that type to the backend plugin, you'll also want to add `'@backstage/plugin-techdocs-backend': patch` here too.

While you're here: if you rename the file with a `techdocs-` prefix, we can likely get this in a little more quickly!